src/java/boa/compiler/transforms/LocalAggregationTransformer.java

```diff
@@ -126,6 +127,7 @@ protected void generateStoreValue(final EmitStatement n) {
 		n.replaceStatement(n,
 			new AssignmentStatement(
 				new Factor(id.clone()),
+				"=",
```

Comment:
This class does not need changed.  Instead, modify the AssignmentStatement class to have a 2 argument constructor that defaults to the '=' operator.